protoc-gen-pbandk/lib/src/jvmTest/kotlin/CodeGeneratorTest.kt

```diff
@@ -30,8 +30,10 @@ class CodeGeneratorTest {
         // Ensure classes and fields were generated successfully
         val message1Clazz = result.classLoader.loadClass("foobar.Message1").kotlin
         val message2Clazz = result.classLoader.loadClass("foobar.Message2").kotlin
+        val message3Clazz = result.classLoader.loadClass("foobar.Variable").kotlin
         message1Clazz.declaredMemberProperties.single { it.name == "intVal" }
         message2Clazz.declaredMemberProperties.single { it.name == "strVal" }
+        message3Clazz.declaredMemberProperties.single { it.name == "name" }
```

Comment:
Rather than checking for the `name` property (which wouldn't be impacted by this bug), you should check that the `VARIABLE_1` and `VARIABLE_CLASS` enum values got named properly in the generated code.

Reply:
See my updated unit test for an example of the correct check.